MYSQL.md

```diff
@@ -167,9 +167,8 @@ $ GRANT ALL ON *.* TO 'mapknitter' WITH GRANT OPTION;
 
 ```
 
-_If you get the error `Access denied for user 'root'@'localhost'" error"`_  
+_If you get the error `Access denied for user 'root'@'localhost'" error"`_  , maybe you forgot your root password or was not prompted when installing. 
```

Comment:
I think no change is needed here.

Reply:
Allright, then I will revert the changes